project_task_stage_state/migrations/16.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2014 Daniel Reis
```

Comment:
Is this a valid attribution?

Reply:
> Is this something we should propose to add in 15.0 version of the module (where the rename seems to have taken place) in oca/project repo?

Yes. We should do.

> If the script is added for 15.0.1.0.0, would it still get triggered when we upgrade the module in the migrated database with the 16.0 module?

Yes. If the 15.0.1.0.0 migration folder is exist in 16.0 module, it will run.

Reply:
I am not clear myself but I thought he has the copy right for this module and I added.